deployment/cdk/opensearch-service-migration/options.md

```diff
@@ -13,7 +13,7 @@ These tables list all CDK context configuration values a user can specify for th
 | migrationConsoleServiceEnabled       | boolean | true                                                                                   | Enable deploying the given service, via a new CloudFormation stack                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
-| migrationAPIEnabled                  | boolean | true                                                                                   | **Experimental** flag for enabling an API on the Migration Console for controlling migration actions                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+| migrationAPIEnabled                  | boolean | true                                                                                   | **Experimental** flag for enabling an API on the Migration Console for controlling migration actions. Target Group will be associated to ECS service which can be used to route requets to the api.                                                                                                                                                                                                                                                                                                                                                                                                                                |
```

Comment:
This seems like too much detail for end user facing documentation, lets remove.

Reply:
Removed, thanks!